src/actions/toolbox.js

```diff
@@ -37,3 +37,19 @@ export function openWorkerToolbox(url: string) {
     }
   };
 }
+
+export function evaluateInConsole(inputString: string) {
+  return async ({ client, getState }: ThunkArgs) => {
+    const frameId = getSelectedFrameId(getState());
+
+    console.log(inputString);
+
+    client.evaluate(`${inputString}.toString()`, { frameId }).then(res => {
+      if (res.exception) {
+        const docLink = res.exceptionDocURL ? `\n${res.exceptionDocURL}` : "";
+        return console.error(res.exceptionMessage, docLink);
```

Comment:
what's this?

Reply:
It's a try to add context to an error message if it is provided via exceptionDocURL.

Reply:
i see. we probably need to evaluate `client.evaluate("console.error(${res.exceptionMessage}, ${docLink})")` or something like that